modules/labs-react/common/stories/Box.stories.mdx

```diff
@@ -0,0 +1,404 @@
+component. These props are useful when a component is a child of a
+[flex container](https://developer.mozilla.org/en-US/docs/Glossary/Flex_Container). Below is a props
+table for flex item props:
+
+<ArgsTable of={BoxFlexItem} />
+
+#### Example
+
+<ExampleCodeBlock code={FlexItem} />
+
+---
+
+### Layout
+
+Layout style props allow you to adjust the display, dimensions, and overflow of a component. These
+props are not connected to our design tokens, but they are often useful for building components.
+Below is a props table for layout props:
+
+<ArgsTable of={BoxLayout} />
+
+#### Example
+
+<ExampleCodeBlock code={Layout} />
+
+---
+
+### Position
+
+Position style props allow you to adjust the position of a component. These props are not connected
+to our design tokens, but they are often useful for building components. There are two types of
+position props: `PositionLogicalProps` and `PositionStandardProps`.
+
+#### Position Logical Props
+
+Position logical props are based on
+[CSS Logical Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties/Basic_concepts).
+They support bidirectionality (LTR and RTL) and will automatically flip the value when a change in
+direction is detected. For example: `insetInlineStart="2px"` would set `left: 2px;` for LTR and
+switch to `right: 2px;` for RTL. You'll want to opt for these properties over standard position
+props, such as `left` and `right` when you need bidrectional styles.
+
+Here's a quick reference for how these props map to LTR and RTL styles:
+
+| Logical Prop       | LTR Style | RTL Style |
+| ------------------ | --------- | --------- |
+| `insetInlineStart` | `left`    | `right`   |
+| `insetInlineEnd`   | `right`   | `left`    |
+
+And below is a props table for the logical props:
+
+<ArgsTable of={BoxPositionLogical} />
+
+---
+
+#### Position Standard Props
+
+Position standard props work the same as position logical props, but have no bidirectional support.
+Bidirectional support will only affect `left` and `right`. Below is a props table for the standard
+props:
+
+<ArgsTable of={BoxPositionStandard} />
+
+#### Example
+
+<ExampleCodeBlock code={Position} />
+
+---
+
+### Space
+
+Space style props allow you to adjust the margin and padding of components. There are two types of
+space props: `SpaceLogicalProps` and `SpaceStandardProps`.
+
+#### Space Logical Props
+
+Space logical props are based on
+[CSS Logical Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties/Basic_concepts).
+They support bidirectionality (LTR and RTL) and will automatically flip the value when a change in
+direction is detected. For example: `marginInlineStart="16px"` would set `margin-left: 16px;` for
+LTR and switch to `margin-right: 16px;` for RTL. You'll want to opt for these properties over
+standard space props, such as `marginLeft` and `marginRight` when you need bidrectional styles.
```

Comment:
```suggestion
standard space props, such as `marginLeft` and `marginRight` when you need bidirectional styles.
```

Reply:
Okay, so I need to work on spelling "bidirectional." 😂 Noted. haha